packages/clerk-js/src/core/resources/Image.ts

```diff
@@ -9,8 +9,12 @@ export class Image extends BaseResource implements ImageResource {
 
   static async create(path: string, body: any = {}): Promise<ImageResource> {
     let fd = body;
-
-    if (body['file'] && typeof body['file'] !== 'string') {
+    let headers;
+    if (typeof body['file'] === 'string') {
+      fd = body['file'];
+      headers = new Headers();
+      headers.set('Content-Type', 'application/octet-stream');
```

Comment:
@panteliselef I would expect this to happen in `fapiClient` thought an option of something

Reply:
I think this is the correct approach. This will reach the fapiClient eventually and the Content-Type header won't be reset there.

https://github.com/clerkinc/javascript/pull/1320/files#diff-71db37a1049d7cb619179c725410d71e5dde304c8502e56117d057c43102b011L176